.github/workflows/build.yaml

```diff
@@ -0,0 +1,21 @@
+name: Go package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.16'
```

Comment:
you have verison 1.20 in go mod - https://github.com/mo-pm888/twitter/blob/38b8c450b7d36f35b046f6c80851f5ff7501f58d/go.mod#L3C4-L3C8
why 1.16 here?